docs/abi-spec.rst

```diff
@@ -81,6 +81,23 @@ It is possible to form tuples of tuples, arrays of tuples and so on.  It is also
 .. note::
     Solidity supports all the types presented above with the same names with the exception of tuples. The ABI tuple type is utilised for encoding Solidity ``structs``.
 
+Mapping Solidity to ABI types
+-----------------------------
+The following Solidity types (left column) are not part of the ABI, and are
+mapped to an existing ABI type (right column):
+
++-------------------------------+-----------------------------------------------------------------------------+
+|      Solidity                 |                                           ABI                               |
++===============================+=============================================================================+
+|:ref:`address payable<address>`|Maps to ``address``.                                                         |
```

Comment:
a construct that does not have a subject is not a sentence and thus should not start with uppercase and should not end with a period ;)

Perhaps the "maps to" could also be in the header?